src/Widgets/GalleryWidget/GalleryWidgetComponent.js

```diff
@@ -16,8 +18,8 @@ Scrivito.provideComponent("GalleryWidget", ({ widget }) => {
 
   const settings = sliderSettings(images);
   return (
-    <div className="slick-gallary-fluid">
-      <Slider {...settings} className="slickslide">
+    <div className="gallery-widget">
+      <Slider {...settings} className="gallery-widget--slick-slide">
```

Comment:
Their is no `gallery-widget--slick-slide` defined. But a `slick-slider` under `gallery-widget`.

Since the default class of a slick slider is `slick-slider` I would simply remove the custom `className`:

```suggestion
      <Slider {...settings}>
```

Please not that `slickslide` and `slick-slider` in the previous version where added to the exact same dom element. So consolidating the two into `.galery-widget .slick-slider` seems to be a good idea.